ACadSharp/IO/Templates/CadInsertTemplate.cs

```diff
@@ -72,6 +74,11 @@ public override void Build(CadDocumentBuilder builder)
 			{
 				insert.Attributes.Seqend = seqend;
 			}
+
+            if (builder is DxfDocumentBuilder)
+            {
+                insert.Rotation *= MathUtils.DegToRad;
```

Comment:
Instead of checking if the builder is a `DxfDocumentBuilder` I would prefer to override the method:
```CSharp
public virtual bool CheckDxfCode(int dxfcode, object value)
```
This method bypassed the mapper when the DxfReader assigns the values, this way we'll keep the assignation in the dxf scope.